### ISSUE TITLE
[cross post] unable to create a cross-post

### ISSUE THREAD

**Issue by diegoberaldin** [diegoberaldin/RaccoonForLemmy#754]

**Comment by diegoberaldin** [comment]
Closed, not reproducible.

**Comment by Mannivu** [comment]
I'm having this issue too. When I try to cross post I send the post the "New post" window closes but nothing happens. No error message, no cross post.

**Comment by diegoberaldin** [comment]
Reopening it... in my case it probably was due to an authentication issue but when I restarted the app to inspect the logs it stopped happening.

**Comment by diegoberaldin** [comment]
@Mannivu I added at least an error indication when the API call fails, which was "accidentally" silenced by default. Let's see if the error indication gives any hint towards resolution, because I haven't been able to reproduce it on purpose. Classic "heisenbug".

**Comment by Mannivu** [comment]
Thanks. I'm not a serial cross-poster so I'm not sure when I'll have another chance to try the feature. But if it happens I'll let you know the outcome.

**Comment by Mannivu** [comment]
Just tested and I was able to cross-post. Not sure why.

**Comment by diegoberaldin** [comment]
I would have bet the issue was either with authentication (expired token) or some bug in the ViewModel state which loses some mandatory piece of information (e.g. the community ID). But when I tried to launch a debug build with the debugger attached to log everything that was sent/received over the network, it started working and have worked ever since, both in dev and in release builds. Surely I introduced some weird bug, I am leaving this issue open so that I remember about it.
